### ISSUE TITLE
Implementation of Not In Logical Operator

### ISSUE THREAD

**Issue by Extuinger** [ddossot/NxBRE#14]
Greetings @ddossot , I have been experimenting on Nxbre from quite some time for my college project. Although it has been amazing journey with it. But I got stuck in implementing a functionality when using If/ElseIf/Else condition. I noticed that there was "**IN**" operator to be used but "**NOT IN**" operator was missing. I tried a way to implement it using Objeclookup and invoking a method of mine own to execute it, but XSL schema that is defined in engine doesn't let me do so.

Basically what I was implementing was

<If>
   <And>
      <Objectlookup id = "CustomMethodNotIn" AssemblyName = "CustomClass">
         <Arguments value = "something"
       </ObjectLookup>
    </And>
</If>

I just wanted to know, if there was some other way to do so or not.
Thanks
